kubernetes/provider.go

```diff
@@ -67,16 +70,11 @@ func Provider() *schema.Provider {
 				DefaultFunc: schema.EnvDefaultFunc("KUBE_CLUSTER_CA_CERT_DATA", ""),
 				Description: "PEM-encoded root certificates bundle for TLS authentication.",
 			},
-			"config_path": {
-				Type:     schema.TypeString,
-				Optional: true,
-				DefaultFunc: schema.MultiEnvDefaultFunc(
-					[]string{
-						"KUBE_CONFIG",
-						"KUBECONFIG",
-					},
-					"~/.kube/config"),
-				Description: "Path to the kube config file, defaults to ~/.kube/config",
+			"config_paths": {
```

Comment:
Now that I've implemented this and I looked at it I still kind of want to support `config_path` too because it feels like a slightly nicer user experience for the average user of this. 

It's slightly clunky and surprising to have set an array if you just have one kubeconfig. It's also kind of clunky to have to set `KUBE_CONFIG_PATHS` to a single path if you're using an environment variable. 

There would be a small overhead to having both in the provider block, but I think it's a more intuitive UX for someone just picking up the tool. Leaving the list option for the subset of users that have more complicated environments.

What do you think? @redeux @alexsomesan @dak1n1

Reply:
I've updated to support both, as per our discussion this week.